lib/common.bash

```diff
@@ -234,6 +234,7 @@ clean_env_ctr()
 	for i in "${containers[@]}"; do
 		task="$(sudo ctr task ls | grep -E "\<${i}\>" | cut -f1 -d" " || true)"
 		[ -n "${task}" ] && sudo ctr task kill "${task}"
+		[ -n "${task}" ] && sudo ctr tasks kill -s SIGKILL "${task}"
```

Comment:
@GabyCT, we have both lines to first try to cleanly kill it, and then force?

Reply:
@cmaf yes it seems that we will need to do that as even if we kill them some containers are left and we will need to force to kill them, I saw some issues of this in the containerd repo